src/core/event.ts

```diff
@@ -67,13 +67,16 @@ export function clientToLocal(
     // <https://bugs.jquery.com/ticket/8523#comment:14>
     // BTW3, In ff, offsetX/offsetY is always 0.
     else if (env.browser.firefox
+        // use offsetX/offsetY for Firefox >= 39
+        // PENDING: consider Firefox for Android and Firefox OS? >= 43
+        && env.browser.version < '39'
```

Comment:
I suppose '39' should be a number here?

Reply:
I'm not sure if the version string in some earlier versions will probably be like `88.0.1`, though I saw the version string in the `navigator.userAgent` is like `88.0` in most of the versions. (Tested it in Firefox 50, 70, 80, 92)

<img src="https://user-images.githubusercontent.com/26999792/132781056-b9e91fb6-065c-43ec-9178-3800fb89f9a5.png" width="400">
<img src="https://user-images.githubusercontent.com/26999792/132781100-5b61a9cb-5358-4880-bab9-63f74e96a10c.png" width="200">

<img src="https://user-images.githubusercontent.com/26999792/132781824-a9a3b176-d1a5-43e1-b7c6-e6cfa3f8bbba.png" width="400">
<img src="https://user-images.githubusercontent.com/26999792/132781831-5ec58fce-cce8-4493-a45f-69a4e93ecb05.png" width="200">

Reply:
Thanks for the explanation. Seems string comparison is better. Let's keep it.